spec.md

```diff
@@ -1367,6 +1373,111 @@ Errors](#service-broker-errors).
 }
 ```
 
+## Fetching a Service Instance
+
+If `"instance_retrievable" :true` is declared for a service in the [Catalog](#catalog-management) endpoint, brokers MUST support this endpoint for all plans of the service.
```

Comment:
If `"instance_retrievable": false`, does that mean the endpoint is should not be supported for ANY plans? Or simply that not ALL plans support it, and it's on a case-by-case basis?

I think the former gives us more information, and since this field is not on a plan-by-plan basis we should make it a sweeping statement where either all plans support it or none do.